src/patternfly/components/Select/select-toggle.hbs

```diff
@@ -37,9 +37,7 @@
 				<span>Select</span>
```

Comment:
Can you fix the indentation in this file? Looks like it's using the wrong tab space value